output/bremen/stops.txt

```diff
@@ -1215,19 +1189,23 @@ stop_id,stop_code,stop_name,stop_desc,stop_lat,stop_lon,zone_id,stop_url,locatio
 987926318,,Lengstraße,,53.5203128,8.5796993,,,,,,
 3644924431,,Am Fischbahnhof,,53.5218548,8.5815929,,,,,,
 27200983,,Am Schaufenster/Gera,,53.521017,8.5851594,,,,,,
-2574919450,,Unnamed train stop.,,53.5350113,8.5993139,,,,,,
-2574919474,,Unnamed train stop.,,53.566754,8.6006485,,,,,,
+2574919474,,Unnamed light_rail stop.,,53.566754,8.6006485,,,,,,
```

Comment:
@mehdisadeghi I didnt notice this before. Would it be possible to stick to our `[Dummy]` flag? that way we have a single identification mechanism for missing values.

Reply:
Yes, sure. I did a lot at once so many thing slipped away!

Reply:
@carocad These are real stops which happen to have no name.